crates/ruff/src/jupyter/schema.rs

```diff
@@ -150,6 +152,7 @@ pub struct CodeCell {
     /// Technically, id isn't required (it's not even present) in schema v4.0 through v4.4, but
     /// it's required in v4.5. Main issue is that pycharm creates notebooks without an id
     /// <https://youtrack.jetbrains.com/issue/PY-59438/Jupyter-notebooks-created-with-PyCharm-are-missing-the-id-field-in-cells-in-the-.ipynb-json>
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Wait, we already have `#[skip_serializing_none]` for other cell types, it's not added to `CodeCell` because the field `execution_count` is [required](https://github.com/jupyter/nbformat/blob/a3e787dccbf629c8cbae46f79e74d53ded8b53a9/nbformat/v4/nbformat.v4.5.schema.json#L197). The solution is to use `serialize_always` _only_ on `id` field of `CodeCell`.

```suggestion
    #[serialize_always]
```